api/Dockerfile

```diff
@@ -1,13 +1,32 @@
-FROM python:3.12.8-alpine3.20 AS build
+FROM python:3.12.10-slim-bookworm AS build
 
 LABEL maintainer="https://github.com/prowler-cloud/api"
 
-# hadolint ignore=DL3018
-RUN apk --no-cache add gcc python3-dev musl-dev linux-headers curl-dev
+ARG POWERSHELL_VERSION=7.5.0
```

Comment:
Should we control this version through environment variable?

Reply:
We could define this version as the default value in case the environment variable is not present. Does this make sense to you?

Reply:
cc: @jfagoagas since you asked Adri to store the version staticly.

Reply:
I can apply this change, @jfagoagas what do you think?

Reply:
I don't know how are we going to manage Powershell dependencies but we'll need to find an automated way and try to have just one source of truth. Regarding the environment variable I think it is good to have it.